ee/codegen/src/utils/casing.ts

```diff
@@ -7,10 +7,60 @@ export function toKebabCase(str: string): string {
     .toLowerCase(); // Convert to lowercase
 }
 
-export function toPascalCase(str: string): string {
-  return str
-    .split(/[-_\s]+/)
-    .map((word) => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase())
+export function createPythonClassName(input: string): string {
```

Comment:
Thanks even more for the gnarly implementation cursor! Lot's of talking back and forth to get this.

Reply:
you can tell when cursor writes a PR by the number of comments between each line

Reply:
So true